code-challenges/401challenges/src/main/java/challenges/stacksandqueues/Queue.java

```diff
@@ -0,0 +1,79 @@
+package challenges.stacksandqueues;
+
+public class Queue {
+    private Node front;
+    private Node back;
+    private int size;
+
+    public Node getFront() {
+        return front;
+    }
+
+    public void setFront(Node front) {
+        this.front = front;
+    }
+
+    public int getSize() {
+        return size;
+    }
+
+    public void setSize(int size) {
+        this.size = size;
+    }
+
+    public Node getBack() {
+        return back;
+    }
+
+    public void setBack(Node back) {
+        this.back = back;
+    }
+
+    public Queue(){
+        this.front = null;
+        this.back = null;
+        this.size = 0;
+    }
+
+    public void enqueue(int value){
+        try
+        {
+            Node node = new Node(value);
+            node.next = null;
```

Comment:
You set every new node's next to null in your constructor so I think this line is redundant